src/components/WalletSelectionComponent.jsx

```diff
@@ -186,7 +187,7 @@ class WalletSelectionComponent extends Component<Props> {
             </Grid>
             <Grid item>
               <Button color='primary'>
-                <ErrorIcon fontSize='small' color='primary' className={classes.faqIcon} />
+                <HelpIcon fontSize='small' color='primary' className={classes.faqIcon} />
```

Comment:
Shouldn't we use https://material.io/tools/icons/?icon=info&style=round ?

Reply:
looks like it's correct according to invision deseign, ignore the previous comment